oximeter/oximeter/schema/http-service.toml

```diff
@@ -0,0 +1,38 @@
+format_version = 1
+
+[target]
+name = "http_service"
+description = "An Oxide HTTP server"
+authz_scope = "fleet"
+versions = [
+    { version = 1, fields = [ "name", "id" ] },
+]
+
+[[metrics]]
+name = "request_latency_histogram"
+description = "Duration for the server to handle a request"
+units = "seconds"
+datum_type = "histogram_f64"
+versions = [
+    { added_in = 1, fields = [ "route", "method", "status_code" ] }
+]
+
+[fields.name]
+type = "string"
+description = "The name of the HTTP server, or program running it"
+
+[fields.id]
+type = "uuid"
+description = "UUID of the HTTP server"
+
+[fields.route]
+type = "string"
+description = "HTTP route in the request"
+
+[fields.method]
+type = "string"
+description = "HTTP method in the request"
+
+[fields.status_code]
+type = "u16"
```

Comment:
Probably not a problem, just confirming: This is changing from an i64 to a u16 -- is that okay?

Reply:
Whoa nelly, good catch. It's definitely _not_ OK today to change the type, we'd get a schema-mismatch error at `oximeter` on existing deployments if we merged this. My mistake.

This raises a slightly larger question actually. Especially at this early stage, I have often wanted to correct things like this. I'm not sure how, since that raises the issue of breaking schema changes. One approach, which I sketched out in a WIP branch, is to include a list of timeseries that we drop entirely as part of a ClickHouse schema update. So for example, if we included this timeseries in that list, we could make changes like this at the expense of losing historical data. I haven't done anything with the branch because it wasn't clear if that would be useful, or the right direction. I do think it would be _useful_ in this case, but I'm still not sure about the second part...

Reply:
The most expedient thing would be to undo this change, decoupling the question of whether / how to remove outdated timeseries from this PR. I'll do that now.

Reply:
Done in 1935da928